src/Microsoft.VisualStudio.Composition/ComposedPart.cs

```diff
@@ -160,6 +173,61 @@ public IEnumerable<ComposedPartDiagnostic> Validate(IReadOnlyDictionary<Type, Ex
             }
         }
 
+        /// <summary>
+        /// Find and removes any invalidated parts binded with an <see cref="ImportCardinality.ExactlyOne"/> from the <see cref="SatisfyingExports"/> dictionary.
```

Comment:
nit: "bound" instead of "binded"

Reply:
Also, describing the implementation details of what this part constitutes as an invalidation probably goes beyond what the caller cares about. It's probably enough for the summary to say "Removes satisfying exports coming from a given set of parts from this part's imports."
When it comes to that, the fact that the argument consists of _invalidated_ parts is not actually relevant to this method. So maybe call this RemoveSatisfyingExports and rename the parameter to `parts`. The doc comment for the parameter could then be changed to "The parts whose exports should not be imported by this part."